### ISSUE TITLE
502 Bad Gateway  Error on web page.

### ISSUE THREAD

**Issue by speed2048** [mrjackwills/adsbdb#33]
When I look up the aircraft information for Aircraft registration: **PR-AYZ** I get 502 Bad Gateway address on the web page.

1. **Goto https://api.adsbdb.com/v0/aircraft/pr-ayz**
2. Wait about 10 - 20 seconds.
3.  Will return an error message.

502 Bad Gateway
mrjackwills.com

**Expected behavior**

To return valid registration information for this Aircraft  

**Desktop (please complete the following information):**
 - Windows 11 using  chrome / edge.

Additional information.

Flight Radar returns the aircraft information but no flight data is returned.

**Comment by mrjackwills** [comment]
Thanks, have put in a temporary fix, but need to look into it properly

**Comment by speed2048** [comment]
Thank you for responding so quickly.

**Comment by fschwarz86** [comment]
The issue is back (at least for me): Getting 502s on all requests, e.g. https://api.adsbdb.com/v0/aircraft/4408de?callsign=AUA173H

**Comment by mrjackwills** [comment]
> The issue is back (at least for me): Getting 502s on all requests, e.g. https://api.adsbdb.com/v0/aircraft/4408de?callsign=AUA173H

Restarted the application, still trying to work out what is causing this issue

**Comment by fschwarz86** [comment]
Wow, that was quick! Thanks a ton, it's working again.

**Comment by mrjackwills** [comment]
> Wow, that was quick! Thanks a ton, it's working again.

Just good timing, happened to be on GitHub when you posted the comment

**Comment by vdbpeter** [comment]
Hi- looks like it's happening again. https://api.adsbdb.com/v0/aircraft/7c6db7
Thanks

**Comment by mrjackwills** [comment]
Yeah that was my fault, server had issues, but it should be resolved now
